src/runtime-rs/crates/hypervisor/ch-config/src/convert.rs

```diff
@@ -481,10 +599,29 @@ where
     Ok(MacAddr { bytes })
 }
 
+// Determine if the specified Share Fs (virtiofs mount) can be safely ignored.
+//
+// Normally, the container rootfs is passed as a virtiofs device.
+// However, if the user requested the container rootfs be passed
+// as a block device (for example if using containerd with the
+// devmapper snapshotter), runtime-rs will pass the container
+// rootfs _twice_: once as a virtiofs device and once as a block
```

Comment:
If in case of devicemapper, the rootfs is being passed twice, this needs to be fixed at some point. I think this needs to be handled in the device manager code. Do we have an issue tracking this?